spec/javascript/components/search/__snapshots__/AdvancedSearchFilter.test.js.snap

```diff
@@ -0,0 +1,77 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`callbacks clears 1`] = `
+[MockFunction] {
+  "calls": Array [
+    Array [],
+  ],
+  "results": Array [
+    Object {
+      "isThrow": false,
+      "value": undefined,
+    },
+  ],
+}
+`;
+
+exports[`callbacks submits 1`] = `
+[MockFunction] {
+  "calls": Array [
+    Array [],
+  ],
+  "results": Array [
+    Object {
+      "isThrow": false,
+      "value": undefined,
+    },
+  ],
+}
+`;
+
+exports[`renders as expected (with query param) 1`] = `
+<div>
+  <input
+    autoComplete="off"
+    className="form-control"
+    id="search_str"
+    name="search"
+    onChange={[MockFunction]}
+    placeholder="Advanced search"
+    type="text"
+    value="query"
```

Comment:
Shouldn't this be `foo`? I'm not seeing `foo` anywhere in this snapshot even though that's the query param...

Reply:
OH RIGHT. I guess this snapshot will change once we integrate with the backend.

Reply:
Actually this value is coming from the test mock in `spec/.../search/utils.js` which defines `advancedSearchProps`. The AdvancedSearchFilter component itself gets props directly, it doesn't care what the actual url querystring is.